charts/kubescape-operator/templates/node-agent/configmap.yaml

```diff
@@ -14,9 +14,27 @@ data:
     {
         "applicationProfileServiceEnabled": {{ $configurations.runtimeObservability }},
         "relevantCVEServiceEnabled": true,
+        "prometheusExporterEnabled": "{{ eq .Values.nodeAgent.config.prometheusExporter "enable" }}",
+        "runtimeDetectionEnabled": "{{ eq .Values.capabilities.runtimeDetection "enable" }}",
```

Comment:
We should fix `runtimeDetectionEnabled` and `networkServiceEnabled` to boolean like other existing boolean configs.


```
	EnableFullPathTracing    bool                      `mapstructure:"fullPathTracingEnabled"`
	EnableApplicationProfile bool                      `mapstructure:"applicationProfileServiceEnabled"`
	EnablePrometheusExporter bool                      `mapstructure:"prometheusExporterEnabled"`
	EnableRuntimeDetection   bool                      `mapstructure:"runtimeDetectionEnabled"`
	EnableNetworkTracing     bool                      `mapstructure:"networkServiceEnabled"`
	EnableRelevancy          bool                      `mapstructure:"relevantCVEServiceEnabled"`
```